### PR TITLE
Update dates, add slide about 2016 django girls, remove brunch

### PR DESCRIPTION
**Pull request by sheins**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,6 +74,12 @@ <h1 style="margin: -70px auto 70px auto;">Kansas City</h1>
 							A total of 4034 incredible women have attended events organized by members of the community.
 						</p>
 					</section>
+					<section id="previous">
+                        <h2>Last Year</h2>
+                        <p>
+                            In 2016, we hosted our first Django Girls in Kansas City. With the help of 16 mentors, we introduced nearly 100 women to web development through django.
```
Comment (tehfedaykin) [inline index.html]: capitalize 'django' ;)

Comment (sheins) [reply index.html]: Hahahaha see we really did need the sad pull request

**Comment by sheins** [comment]
@tehfedaykin this has been updated

**Review by tehfedaykin — APPROVED** [review]
sweet

**Pull request merged**